src/tsUnionTypes/task2.ts

```diff
@@ -0,0 +1,6 @@
+const fn = (argument?: string | undefined) =>
+  typeof argument === "string" ? argument.match(/[A-Z]/g)?.length : null;
```

Comment:
sąlygą ` typeof argument === "string"`galima būtų gerokai sutrumpinti, turint omeny, kad yra galimi tik du argument tipai - string arba undefined. Galima apsieiti be konkretaus tipo tikrinimo.

Ką grąžina funkcija jeigu argumentas yra stringas, bet neturi didžiųjų raidžių? pvz "aaaa"?